build.sbt

```diff
@@ -179,6 +181,16 @@ lazy val reporter =
     )
     .dependsOn(domain, serializer)
 
+lazy val buildInfo =
```

Comment:
How come we have a separate build target for `buildInfo` instead of just including the buildInfo stuff inside of the `plugin` module and enabling the `BuildInfoPlugin` there?

Reply:
Aha, yes I suppose we could. I did this because the `BuildInfo` was only needed in the `Test` scope and there's currently no way to configure that (see https://github.com/sbt/sbt-buildinfo/issues/186). But I guess here if we "leak" the `BuildInfo" as part of the shipped plugin, it's probably not a big deal?